PluginCode/src/main/java/smr/cs/ualberta/libcomp/DatabaseAccess.java

```diff
@@ -37,6 +42,11 @@
     private int userid = 0;
     private String filePath = PathManager.getPluginsPath()+"\\Library_Comparison\\lib";
     private String feedbackUrllink = "https://smr.cs.ualberta.ca/comparelibraries/api/pluginfeedback/";
+    private String linkURL = "https://smr.cs.ualberta.ca/comparelibraries/api/metrics/?format=json&latestdate=";
+    private String librariesURL = "https://smr.cs.ualberta.ca/comparelibraries/api/libraries/?format=json";
+    private String chartsURL = "https://smr.cs.ualberta.ca/comparelibraries/api/charts/?format=json";
+    private String metricsURL = "https://smr.cs.ualberta.ca/comparelibraries/api/metrics/?format=json&latestdate=";
+    private String updateUrllink = "https://smr.cs.ualberta.ca/comparelibraries/api/pluginusers/";
```

Comment:
@relhajj these are still hard-coded domains, not in a config file. Given time, we can ignore it but this isn't what we agreed on in Issue #36